CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```

Comment:
So this changelog doesn't quite adhere to keepachangelog.

The format here is a bit bespoke at this point. 

Unless we want to go through the trouble of migrating the format, I would only mention that we follow semver.

Reply:
Updated.